frontend/src/views/auth/Login.vue

```diff
@@ -24,7 +24,13 @@
       </div>
     </v-alert>
     <v-alert v-if="error" outlined text border="left" type="error">
-      {{ error }}
+      <span class="d-block">{{ error }}</span>
+      <span class="d-block mt-1"
```

Comment:
I was wondering, if we should also include the password reset link here. I know it can be found further down again. But know we very prominently print the resend-link, while the reason for invalid credentials could also be a wrong password.
- Email not yet confirmed? Resend activation email
- Password forgotten? Reset password

Or can we distinguish from the error message, which case it is?

Reply:
done